### DISCUSSION TITLE
Root File Testing

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#92] [Q&A] [answered]
Is there a way to test that the root file is setup correctly? I recently had a violation that my TXT file was not in compliance. I have since added the TXT file to the root folder of my website and ensured the name of the file is in accordance with CMS guidelines. Before I send my response to CMS that it is completed, I wanted to make sure the TXT file is working.

**Comment by rlarsonat** [comment]
I attended a webinar earlier this year that gave a way to test.  You should be able to type into your web address "www.yourhospital.com/cms-hpt.txt" Obviously, you would substitute the your hospital with your hospital's web details or whatever follows the @ in your email.  For instance, our facility is www.amberwellhealth.org/cms-hpt.txt

**Comment by RidenourK** [comment, marked as answer]
@vduffel99. Thank you for your question.

Guidance on how to generate and where to place the TXT file, along with the TXT File Generator itself, is available on the HPT Tools Site here: https://cmsgov.github.io/hpt-tool/txt-generator/. 

Frequently Asked Questions on the TXT File may be found here: https://www.cms.gov/files/document/hospital-price-transparency-txt-file-frequently-asked-questions-faqs.pdf.
